### PR TITLE
[FONTVIEW] Update German (de-DE) translation

### PR DESCRIPTION
**Pull request by hpreusse**
## Purpose
String translation in de-DE.rc

### CODE REVIEW

**Review by s0mt0chukwu — COMMENTED** [review]
I wish I could read German and be a Nazi

File: base/applications/fontview/lang/de-DE.rc [hunk 1]
```diff
@@ -10,19 +10,19 @@ BEGIN
...
     IDS_ERROR_NOCLASS "Fehler beim Initialisieren der Fensterklasse."
-    IDS_ERROR_ISINSTALLED "This font is already installed!" 
-    IDS_ERROR_FONTCPY "Failed to copy the font file!"
-    IDS_ERROR_OPENKEY "Failed to open the fonts key!"
-    IDS_ERROR_REGISTER "Failed to register the new font!"
-    IDS_SUCCESS "Success"
-    IDS_COMPLETED "Font installation completed."
-    IDS_FILTER_LIST "All Supported Fonts (*.fon;*.fnt;*.ttf;*.ttc;*.otf;*.otc)\0*.fon;*.fnt;*.ttf;*.ttc;*.otf;*.otc\0\
+    IDS_ERROR_ISINSTALLED "Die Schriftart ist schon installiert!" 
+    IDS_ERROR_FONTCPY "Kopieren der Schriftartdatei fehl geschlagen!"
```
Comment (JoachimHenze) [inline base/applications/fontview/lang/de-DE.rc:14]:
```suggestion
    IDS_ERROR_FONTCPY "Kopieren der Schriftartdatei fehlgeschlagen!"
```

File: base/applications/fontview/lang/de-DE.rc [hunk 2]
```diff
@@ -10,19 +10,19 @@ BEGIN
...
+    IDS_ERROR_OPENKEY "Öffnen des Schriftart Keys fehl geschlagen!"
```
Comment (JoachimHenze) [inline base/applications/fontview/lang/de-DE.rc:15]:
```suggestion
    IDS_ERROR_OPENKEY "Öffnen des Schriftart Keys fehlgeschlagen!"
```

File: base/applications/fontview/lang/de-DE.rc [hunk 3]
```diff
@@ -10,19 +10,19 @@ BEGIN
...
+    IDS_ERROR_REGISTER "Registrierung der neuen Schriftart fehl geschlagen!"
```
Comment (JoachimHenze) [inline base/applications/fontview/lang/de-DE.rc:16]:
```suggestion
    IDS_ERROR_REGISTER "Registrierung der neuen Schriftart fehlgeschlagen!"
```

File: base/applications/fontview/lang/de-DE.rc [hunk 4]
```diff
@@ -10,19 +10,19 @@ BEGIN
...
+    IDS_FILTER_LIST "Alle unterstützten Schriftarten (*.fon;*.fnt;*.ttf;*.ttc;*.otf;*.otc)\0*.fon;*.fnt;*.ttf;*.ttc;*.otf;*.otc\0\
 Font File (*.fon;*.fnt)\0*.fon;*.fnt\0\
 TrueType Font (*.ttf)\0*.ttf\0\
 TrueType Font Collection (*.ttc)\0*.ttc\0\
 OpenType Font (*.otf)\0*.otf\0\
 OpenType Font Collection (*.otc)\0*.otc\0\
 All Files (*.*)\0*.*\0"
-    IDS_PREVIOUS "< P&revious"
-    IDS_NEXT "&Next >"
+    IDS_PREVIOUS "< V&orheriges"
```
Comment (JoachimHenze) [inline base/applications/fontview/lang/de-DE.rc:26]: Can you tell me how to reach that button in Windows fontview? I do want to quickly double-check whether &o is the correct accelerator here.

**Review by JoachimHenze — CHANGES_REQUESTED** [review]
Thank you for your contribution.
Just 3 small suggestions that you can accept by just clicking the button in github webview.
And one question.

File: base/applications/fontview/lang/de-DE.rc [hunk 5]
```diff
@@ -10,19 +10,19 @@ BEGIN
...
+    IDS_ERROR_REGISTER "Registrierung der neuen Schriftart fehlgeschlagen!"
+    IDS_SUCCESS "Erfolg"
+    IDS_COMPLETED "Installation der Schriftarten vollständig."
+    IDS_FILTER_LIST "Alle unterstützten Schriftarten (*.fon;*.fnt;*.ttf;*.ttc;*.otf;*.otc)\0*.fon;*.fnt;*.ttf;*.ttc;*.otf;*.otc\0\
 Font File (*.fon;*.fnt)\0*.fon;*.fnt\0\
 TrueType Font (*.ttf)\0*.ttf\0\
 TrueType Font Collection (*.ttc)\0*.ttc\0\
 OpenType Font (*.otf)\0*.otf\0\
 OpenType Font Collection (*.otc)\0*.otc\0\
 All Files (*.*)\0*.*\0"
```
Comment (binarymaster) [inline base/applications/fontview/lang/de-DE.rc:25]: You may want to translate these also, or at least the last "All Files" one.

Comment (hpreusse) [reply base/applications/fontview/lang/de-DE.rc]: Thanks. I wasn't aware that these words are part of the translatebly strings.

**Review by JoachimHenze — APPROVED** [review]

**Pull request merged**